inspirehep/modules/authors/bundles.py

```diff
@@ -23,10 +23,33 @@
 """Bundles for author forms."""
 
 from flask_assets import Bundle
+from invenio_assets import NpmBundle
 
-css = Bundle(
+update_css = Bundle(
     "scss/authors/authors-update-form.scss",
     output='gen/inspire-author-update.%(version)s.css',
     depends='scss/forms/*.scss',
     filters="scss, cleancss"
 )
+
+profile_css = Bundle(
+    "scss/authors/authors-profile.scss",
+    output='gen/authors.%(version)s.css',
+    depends='scss/forms/*.scss',
+    filters="scss, cleancss"
+)
+
+js = NpmBundle(
+    'js/authors/app.js',
+    'js/author_settings.js',
+    filters='requirejs',
+    depends=(
+        'js/authors/*.js'
+    ),
+    output="gen/authors.%(version)s.js",
+    npm={
+        "jquery": "~1.9.1",
```

Comment:
No need for jquery and angular to be added here as they are already present in other `npm` sections in other bundles.